app/controllers/errorHandling/ErrorResponses.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.errorHandling
+
+import play.api.libs.json.{JsValue, Json, Writes}
+import play.api.mvc.Result
+import play.api.mvc.Results.Status
+
+sealed abstract class ErrorResponse(val httpStatusCode: Int, val errorCode: String, val message: String) {
```

Comment:
Still need this now you're just getting the HttpResponse back? Can pop over for a chat if you want.

Reply:
done